base/applications/mspaint/dib.cpp

```diff
@@ -69,6 +71,13 @@ SaveDIBToFile(HBITMAP hBitmap, LPTSTR FileName, HDC hDC, LPSYSTEMTIME time, int
     // TODO: update hRes and vRes
 
     CloseHandle(hFile);
+
+    if (bAlreadyExists)
+        SHChangeNotify(SHCNE_UPDATEITEM, SHCNF_PATHW, FileName, NULL);
```

Comment:
This cannot possibly be right. Every application that modifies files does not have to call this function. The filesystem is responsible for that. Please test this on Windows / against Windows's mspaint

Reply:
Reverted.